### ISSUE TITLE
WinVi 64-bit port

### ISSUE THREAD

**Issue by galvanopus** [FigBug/WinVi#1]
Have you abandoned porting WinVi to 64-bit idea or not?

**Comment by FigBug** [comment]
Yeah, pretty much. Haven't had time to look into this lately. It was generating an invalid binary, never could figure out why.

**Comment by galvanopus** [comment]
Oh, I see. Ok, will track this issue, may be you will have time in the near future.
